### PR TITLE
Dev -> Prod 10.18.0

### PR DESCRIPTION
**Pull request by jkonrath-postman**
TW-1344 doc MQTT #5046
TW-1294 doc Scratch Pad EOL #5052
TW-1322 billing section #5048
MARKENG-2715 update prices on Auto Flex Page #5053
TW-1328 Postbot changes for 9/15 #5041
TW-1303 Doc new Proxy FTUX for Windows #5025
TW-1304 Doc export results of performance test to a file #5022
TW-1335 Collection permissions in Partner Workspaces #5030
TW-1324 Restrict team workspaces #5035
TW-1295 Update workspace template images and instructions #5012
TW-916 Secret Scanner updates #5049
TW-1309 Update Managing API keys #5027
TW-1258 Private API Network features #5028
TW-1343 domain verification updates #5047
TW-1302 - Doc select schema file instead of folder for Git integration #5018
TW-1305 Note purging of old monitor run logs #5023
TW-1306 Doc fix for truncating leading zeros from data files #5026
TW-1110 Autosave api client #4845
TW-1342 Flows for basic users #5032

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md [hunk 1]
```diff
@@ -40,76 +40,79 @@ The _Private API Network_ provides a [central directory](https://www.postman.com
...
+Then you can filter elements with the following options:
 
-The **All APIs, collections and workspaces** section enables you to search, sort, and filter elements with the following options:
+* To filter by element type, select **Element type**, then select **Workspaces**, **Collections**, **APIs**, **Request**, or **Folder**.
+* To filter based on element tags, select **Search tags**, then select the tags added to collections, APIs, or workspaces. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#tagging-a-workspace).
+* To filter by an element's location in a folder in the left sidebar, select **Location**, then select the folder or subfolder. When you filter by a folder, your search results return elements in the folder and its subfolders. To filter by all elements in the network, including elements not organized in folders, don't filter by an element's location.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md:93]:
```suggestion
* To filter by an element's location in a folder in the left sidebar, select **Folder**, then select the folder or subfolder. When you filter by a folder, your search results return elements in the folder and its subfolders. To filter by all elements in the network, including elements not organized in folders, don't filter by an element's location.
```

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collections/using-collections.md [hunk 2]
```diff
@@ -73,6 +73,8 @@ To customize and configure a new collection, do the following:
 1. Select **Tests** to define a [test script](/docs/writing-scripts/test-scripts/) for your collection, which will run after a response is received.
 1. Select **Variables** to define values for [collection variables](/docs/sending-requests/variables/) to share across all requests in the collection.
 
+> You can turn on autosave to automatically save your changes to collections, requests, and environments. Learn more about [autosave](/docs/getting-started/settings/#application).
```
Comment (mark-anthony-postman) [inline src/pages/docs/collections/using-collections.md:76]:
```suggestion
> You can turn on autosave to automatically save your changes to collections, requests, and environments. Learn more about [autosave](/docs/getting-started/installation/settings/#application).
```

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collections/using-collections.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/collections/using-collections.md:76]: Thanks for catching this!

File: src/pages/docs/sending-requests/managing-environments.md [hunk 3]
```diff
@@ -78,6 +78,8 @@ To create a new environment, do the following:
 
 1. Select __Save__ to save any environment variables you have added.
 
+> You can turn on autosave to automatically save your changes to collections, requests, and environments. Learn more about [autosave](/docs/getting-started/settings/#application).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/sending-requests/managing-environments.md:81]:
```suggestion
> You can turn on autosave to automatically save your changes to collections, requests, and environments. Learn more about [autosave](/docs/getting-started/installation/settings/#application).
```

File: src/pages/docs/sending-requests/requests.md [hunk 4]
```diff
@@ -65,14 +65,16 @@ Your requests can include multiple details determining the data Postman will sen
...
-<img alt="Create new request" src="https://assets.postman.com/postman-docs/v10/create-new-http-v10-3.jpg" width="500px"/>
+<img alt="Create new request" src="https://assets.postman.com/postman-docs/v10/create-new-http-v10-4.jpg" width="500px"/>
 
 Select **Save** to create your request. You can give your request a name and description, and choose or create a collection to save it in.
 
 Once your new tab is open, you can specify the details you need for your request.
 
 ![New Request](https://assets.postman.com/postman-docs/v10/empty-request-v10.jpg)
 
+> You can turn on autosave to automatically save your changes to collections, requests, and environments. Learn more about [autosave](/docs/getting-started/settings/#application).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/sending-requests/requests.md:76]:
```suggestion
> You can turn on autosave to automatically save your changes to collections, requests, and environments. Learn more about [autosave](/docs/getting-started/installation/settings/#application).
```

**Review by akinard-postman — APPROVED** [review]

**Pull request merged**